### ISSUE TITLE
Today a Dutch frigate was bought in  landlocked Amsterdam (AI), spawned in a city bordering poulder and shot my unit two LAND tiles away, shot to death in fact 🤦‍♂️

### ISSUE THREAD

**Issue by michaelgo361** [LoneGazebo/Community-Patch-DLL#9547]
_1. Mod Version (X.Y.Z). Current Version: 3.0.4_

_2. DirectX Version_

_3. List of Other Mods_

_4. Describe the Issue_

______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

_8. Steps to reproduce the Issue (Optional)_

_9. Screenshots of the Issue (Optional)_

**Comment by jayeye2011** [comment]
I was watching @michaelgo361 video, I think the death occured between video episodes but he talked about what happened at the beginning of the next episode
https://www.youtube.com/watch?v=NZ6QOu5Q8t4&t=20s
and a minute later he was able to kill a Dutch frigate using his knight
https://www.youtube.com/watch?v=NZ6QOu5Q8t4&t=60s
Yeah, pretty entertaining. 
@reckless Maybe you need to put an exception for ocean tile adjacent to city to prevent poulders being build that cut off direct ocean access for the city.

**Comment by jayeye2011** [comment]
Screenshot of @michaelgo361 video's poulders shows poulder #1 built following 3 land tile rule. #2 poulder was then built next to city only because #1 poulder exists, #3 poulder was then built only because #2 poulder exists. In reality, only poulder #1 should ever be built as this would have maintained a visible blue water ocean link to the city.
![Poulder screenshot](https://user-images.githubusercontent.com/69315126/207852815-9b365e0a-c69c-4b12-8af2-e4311896ee81.jpg)

**Comment by jayeye2011** [comment]
@LessRekkless Alternatively, if a polder allows canal passage for naval units like forts, citadels, cities do, it also does not allow either melee or ranged naval unit to attack while on the polder tile, and also does not provide any defensive bonus unlike forts, citadels, and cities.

**Comment by azum4roll** [comment]
It's still a water tile. Ships should be allowed to do whatever they can in water.

**Comment by michaelgo361** [comment]
Can you not ever leave things as they are if they aren't broken or in some way totally unacceptable. Leastways don't make changes that create more problems than they are worth. Honestly, some things just get me going in the worst way. Leastways things should be tested. Now you got a bloody forum where every self important .... wants a million things changed which will create ever more problems and force people to constantly update mods/maps, whatever... Just saying, make important, necessary changes. The rest should be mods or created in such a way that people can add it if they like and not if they don't. And please don't think that the great proposal forum is a real thing where people can feel justified because they got their proposal the required votes. They mainly do because most people just can't be bothered with the whole setup. It's also how a lot of unwanted elements gain political power all over the world. It is a sad fact.

**Comment by jayeye2011** [comment]
> It's still a water tile. Ships should be allowed to do whatever they can in water.

I don't believe ships are traversing polders in the Netherlands in real life except by sluices or canals in the polder which I believe was one of the justifications for the adoption of the polder proposal
https://www.merriam-webster.com/dictionary/polder
https://www.fao.org/family-farming/detail/en/c/283291/

**Comment by LessRekkless** [comment]
> > It's still a water tile. Ships should be allowed to do whatever they can in water.
> 
> I don't believe ships are traversing polders in the Netherlands in real life except by sluices or canals in the polder which I believe was one of the justifications for the adoption of the polder proposal Edit: For all intents in purposes in real life, polders are dry land (although artificially kept dry) not water ways. https://www.merriam-webster.com/dictionary/polder https://www.fao.org/family-farming/detail/en/c/283291/

Doesn't matter.  The polder on water tiles should be functioning as water that land units can walk on. Nothing else.

**Comment by LessRekkless** [comment]
The only issue here is that the polder improvement unexpectedly managed to increase the sight height of the tile.

**Comment by azum4roll** [comment]
Is this still a thing? I've seen reports that a ranged unit on a water polder does have the correct altitude and cannot shoot inland.
